packages/boot/src/__tests__/acceptance/model-api.booter.acceptance.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ApplicationConfig} from '@loopback/core';
+import {juggler, RepositoryMixin} from '@loopback/repository';
+import {RestApplication} from '@loopback/rest';
+import {
+  createRestAppClient,
+  expect,
+  givenHttpServerConfig,
+  TestSandbox,
+} from '@loopback/testlab';
+import {resolve} from 'path';
+import {BootMixin, ModelApiBooter} from '../..';
+import {StubModelApiBuilderComponent} from '../fixtures/stub-model-api-builder';
+
+describe('rest booter acceptance tests', () => {
+  let app: BooterApp;
+  const SANDBOX_PATH = resolve(__dirname, '../../.sandbox');
+  const sandbox = new TestSandbox(SANDBOX_PATH);
+
+  beforeEach('reset sandbox', () => sandbox.reset());
+  beforeEach(givenAppWithDataSource);
+
+  afterEach(stopApp);
+
+  it('gets simple controller path', async () => {
```

Comment:
Since these tests are supposed to use a stub builder, I wasn't sure how to make sure it was properly building the repo and controller from the model class without doing what [`CrudRestApiBuilder`](https://github.com/strongloop/loopback-next/pull/3617/files?short_path=1ac7202#diff-9115bcf4d0620110bf5141ec01ef5c0bR31) was doing, so any suggestions would be greatly appreciated 🙇‍♀